recipe/meta.yaml

```diff
@@ -1,32 +1,32 @@
 {% set name = "spaghetti" %}
-{% set version = "1.6.10" %}
-{% set sha256 = "c4f8c61e055f82ed8495e8f3c5c34c629cab0c4b62e5ef643a0ec117b8810e9b" %}
+{% set version = "1.7.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spaghetti-{{ version }}.tar.gz
```

Comment:
@ocefpaf Is it not best practice to set variables for `name`, `version`, and `sha` like I had set previously?

Reply:
Only name b/c that is reused, the rest are single use variables so no need.